.circleci/config.yml

```diff
@@ -527,45 +624,88 @@ workflows:
             branches:
               only: /.*/
 
+      - build_images:
+          context: "OC Common"
+          requires:
+            - go_lint
+            - push_base
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              only: /.*/
+
+      - push_base:
```

Comment:
@fredbi If the base images' tag `latest` are based only on master, should it not be triggered only when the master is updated?